site/_includes/partials/devtools/zh/whats-new.md

```diff
@@ -1,12 +1,12 @@
 ## 更多 DevTools 的功能 {: #whats-new }
 欲查询完整的 DevTools 已发布的功能，请参考 <a href="/tags/new-in-devtools/" translate="no">What's New In DevTools</a> 英文系列。以下是部分已翻译成中文的内容。
 
-<!-- ### Chrome 97 {: #chrome97 }
-* [Preview feature: New Recorder panel](/zh/blog/new-in-devtools-97/#recorder)
-* [Refresh device list in Device Mode](/zh/blog/new-in-devtools-97/#device)
-* [Autocomplete with Edit as HTML](/zh/blog/new-in-devtools-97/#code-completion)
-* [Improved code debugging experience](/zh/blog/new-in-devtools-97/#debugging)
-* [Syncing DevTools settings across devices](/zh/blog/new-in-devtools-97/#sync) -->
+### Chrome 97 {: #chrome97 }
+* [预览特性：新的“录制器”面板](/zh/blog/new-in-devtools-97/#recorder)
+* [多设备模式中更新的设备列表](/zh/blog/new-in-devtools-97/#device)
+* [以 HTML 格式修改时自动补全](/zh/blog/new-in-devtools-97/#code-completion)
```

Comment:
修改时用 HTML 格式自动补全

Reply:
I'd say "修改时用 HTML 格式自动补全" is a bit confusing, because what we try to say here is the ability to auto-complete (HTML code) while editing the DOM tree as HTML. Maybe "修改 HTML 时自动补全" is more clear?

Reply:
Okay